packages/components/src/RandomAreaPlotAnimation.tsx

```diff
@@ -1,27 +1,26 @@
 /* eslint-disable react-hooks/exhaustive-deps */
 /* eslint-disable react/display-name */
 
-import React, { useEffect, useState, useRef } from 'react';
+import React, { useEffect, useState, useRef, useMemo } from 'react';
 import debounce from 'lodash.debounce';
 import { assertNotNull } from '@deephaven/utils';
 import './RandomAreaPlotAnimation.scss';
-import ThemeExport from './ThemeExport';
+import { getLoginAnimationThemeColors } from './theme';
 
 const VOLATILITY = 0.025; // how spikey the data gets
 const LOW = 0.9;
 const HIGH = 0.1;
 const START_Y = 0.618;
-const FOREGROUND = ThemeExport['primary-dark'];
 const GRID_SIZE = 80;
-const GRID_COLOR = ThemeExport['gray-800'];
-const BACKGROUND = ThemeExport['gray-900'];
 const PATTERN = { SIZE: 8, DOT_SIZE: 2, FILL_OPACITY: 0.25 };
 const PX_PER_SECOND = 18;
 const RESIZE_DEBOUNCE = 250;
 const INTERACTION_TIMEOUT = 60 * 1000;
 
 // Draw a background canvas, paint it with a fun chart looking animation
 const RandomAreaPlotAnimation = React.memo(() => {
+  const themeColors = useMemo(getLoginAnimationThemeColors, []);
```

Comment:
There's no way to change the theme on the login screen, but regardless - shouldn't this be dependent on the active theme anyway? In case we ever do add a changer to the login screen or we use this animation somewhere else or something.

Reply:
I fixed this so that the animation now responds to theme changes. Added a sample to styleguide as well